.travis.yml

```diff
@@ -0,0 +1,20 @@
+language: php
+
+php:
+  - 5.6
+  - 7.0
+  - 7.1
+
+env:
+  matrix:
+    - PREFER_LOWEST="--prefer-lowest"
+    - PREFER_LOWEST=""
+
+before_script:
+  - composer update --prefer-source $PREFER_LOWEST 
+
+script:
+  - vendor/bin/phpunit
+
+after_script:
+  - php vendor/bin/coveralls -v
```

Comment:
Right, I could not find composer.json changes here, probably because PR is so big